lib/tlCore/Audio.cpp

```diff
@@ -396,10 +474,12 @@ namespace tl
             {
                 auto item = in.front();
                 in.pop_front();
-                std::memcpy(out, item->getData(), byteCount - size);
-                const size_t newItemByteCount = item->getByteCount() - (byteCount - size);
-                auto newItem = audio::Audio::create(item->getInfo(), newItemByteCount / item->getInfo().getByteCount());
-                std::memcpy(newItem->getData(), item->getData() + byteCount - size, newItemByteCount);
+                const size_t itemByteCount = byteCount - size;
```

Comment:
What are these changes for?

Reply:
Nothing in particular.  I removed some duplication of the substraction done a couple of times.  I can revert it back to what it was.

Reply:
Yes please, for complex changes like this only include the relevant code in the PR, it helps make it easier to review. I'm fine with refactoring changes but as separate PRs.